packages/framework/client-logger/app-insights-logger/src/fluidAppInsightsLogger.ts

```diff
@@ -23,18 +81,66 @@ export class FluidAppInsightsLogger implements ITelemetryBaseLogger {
 	 * or else logging will not occur.
 	 */
 	private readonly baseLoggingClient: ApplicationInsights;
+	private readonly config: FluidAppInsightsLoggerConfig;
+	private readonly filters: TelemetryFilter[] = [];
 
-	public constructor(client: ApplicationInsights) {
+	public constructor(client: ApplicationInsights, config?: FluidAppInsightsLoggerConfig) {
 		this.baseLoggingClient = client;
+		if (config === undefined) {
+			this.config = {
+				filterConfig: {
+					mode: "exclusive",
+					filters: [],
+				},
+			};
+		} else {
+			this.config = config;
+			this.initializeFiltersFromConfig(config);
+		}
+	}
+
+	private initializeFiltersFromConfig(config: FluidAppInsightsLoggerConfig): void {
+		if (config.filterConfig.filters === undefined) {
+			return;
+		}
+
+		for (const filter of config.filterConfig.filters) {
+			this.filters.push(filter);
+		}
+
+		// Sort filters using priority from low to high
+		this.filters.sort((a, b) => a.priority - b.priority);
 	}
```

Comment:
If I'm right on the other comment and priority goes away, I think this function can also go away and we can setup the filters directly in the constructor when it's necessary.

Reply:
Applied your suggestions.